examples/transfers/cancellations_test.go

```diff
@@ -0,0 +1,53 @@
+package transfers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/moovfinancial/moov-go/pkg/moov"
+)
+
+func ExampleClient_CancelTransfer() {
+	mc, err := moov.NewClient()
+	if err != nil {
+		fmt.Errorf("new Moov client: %v", err)
+		return
+	}
+
+	var (
+		ctx              = context.Background()
+		partnerAccountID = "00000000-00000000-00000000-00000000"
+		transferID       = "00000000-00000000-00000000-00000000"
+	)
+
+	cancellation, err := mc.CancelTransfer(ctx, partnerAccountID, transferID)
+	if err != nil {
+		fmt.Printf("cancelling transfer: %v", err)
+		return
+	}
+
+	fmt.Printf("Created cancellation: %+v", cancellation)
+}
+
+func ExampleClient_GetCancellation() {
```

Comment:
combine w/ above example?

Reply:
Don't think there's any benefit of combining -- we'd want to keep separate since it's a separate method.